src/main/java/mate/academy/springboot/datajpa/service/mapper/DtoMapper.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.springboot.datajpa.service.mapper;
+
+public interface DtoMapper<D, T, S> {
+    D toModel(T requestDto);
+
+    S toDto(D element);
```

Comment:
it is not really readable, so I'd suggest changing the naming of the types to something more handy